Loop/Views/SettingsView.swift

```diff
@@ -21,6 +21,8 @@ public struct SettingsView: View, HorizontalSizeClassOverride {
     @State var showCGMChooser: Bool = false
     @State var showServiceChooser: Bool = false
     @State var showTherapySettings: Bool = false
+    @State var showDeletePumpDataAlert = false
```

Comment:
[tiny nit] with a verb at the front these read like actions or functions, but they're just bools. `therapySettingsIsPresented`